tests/qiskit_backend_test.py

```diff
@@ -107,15 +106,6 @@ def test_unitary() -> None:
         assert np.allclose(u, u2)
 
 
-def test_cancel() -> None:
```

Comment:
Why was this test deleted?

Reply:
Awkward. Bad merge, although I don't understand how. Will revert.